### ISSUE TITLE
Fail on composing one-time message for push notification

### ISSUE THREAD

**Issue by VicSoft** [Countly/countly-server#1115]
Hi everyone,

recently i've installed a community version of countly-server on my ouwn server to test some push notifications in my mobile application for Android, but when I'm start compose a message to send as a push notification the 3th step of view is not showing message area to fill content. Anyone knows what causes this error?

NOTE: when that happens I see this message at console from browser: TypeError: S.PERS_OPTS is undefined

### Expected Behavior
Send a simple push notification for installed user

### Current Behavior
Screen for compose message doesn't load

### Screenshots (if appropriate):
![Screen Shot 2019-11-25 at 23 33 42](https://user-images.githubusercontent.com/5003578/69592298-4ac30680-0fd4-11ea-8d68-9b7f90418047.png)

I'll try to fix this issue, because I didn't found anything about this behaviour... but I hope anybody had seen this.

tks.

**Comment by iartem** [comment]
It's a known issue, fixed a while ago. Just replace your countly folder with git master and run `sudo grunt dist-all` in countly folder and then `sudo countly restart`.

**Comment by alitanirlar** [comment]
I have installed a clean version last week and reinstall again still same issue.

**Comment by iartem** [comment]
What do you mean by "clean version"?

**Comment by hellais** [comment]
I also noticed this issue when following the official community edition install instructions.

In order to fix it I followed the advice from @iartem and did (I was in the root of where the countly install was done, in my case the user home):
```
git clone https://github.com/Countly/countly-server.git
cd countly
sudo cp -rf ../countly-server/* .
sudo grunt dist-all
sudo countly restart
``` 

After this I had to do a force refresh of the browser to make it work as expected.

@iartem are there plans to make a new release of the countly community edition which includes this important bug fix?
